src/caret_analyze/infra/lttng/records_provider_lttng.py

```diff
@@ -486,6 +486,17 @@ def timer_records(self, timer: TimerStructValue) -> RecordsInterface:
 
         timer_events_factory = self._lttng.create_timer_events_factory(timer_lttng_cb)
         callback_records = self.callback_records(timer.callback)
+        if len(callback_records) == 0:
+            records = RecordsFactory.create_instance(
+                None,
+                columns=[
+                    ColumnValue(COLUMN_NAME.TIMER_EVENT_TIMESTAMP),
+                    ColumnValue(COLUMN_NAME.CALLBACK_START_TIMESTAMP),
+                    ColumnValue(COLUMN_NAME.CALLBACK_END_TIMESTAMP),
```

Comment:
Columns are not in alphabetical order.
If there is no special reason for this, it should be corrected.

Reply:
The order of these columns is not alphabetical.
For columns representing system time, they are in chronological order from left to right.

1. timer wake up
2. callback start
callback end